### PR TITLE
break namespace and word in toc

### PR DESCRIPTION
**Pull request by superyyrrzz**
#1496 
Sample: http://superyyrrzz.github.io/DOCFX/api/Microsoft.DocAsCode.html

### CODE REVIEW

File: src/docfx.website.themes/default/styles/docfx.js [hunk 1]
```diff
@@ -483,6 +483,10 @@ $(function () {
           if (util.getAbsolutePath(e.href) === currentHref) {
             $(e).addClass(active);
           }
+
+          $(e).text(function(index, text) {
```
Comment (hellosnow) [inline src/docfx.website.themes/default/styles/docfx.js]: >index [](start = 29, length = 5)

q: index not used?

Comment (superyyrrzz) [inline src/docfx.website.themes/default/styles/docfx.js]: yes, because .text() requires 2 parameters
http://api.jquery.com/text/#text-function

**Comment by qinezh** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Comment by JacquesBarbe** [comment]
Why doesn't it work with `statidoc` template?
https://github.com/dotnet/docfx/issues/2653

**Pull request merged**